src/compounds/Item/CardBody.jsx

```diff
@@ -38,7 +39,10 @@ const CardBody = ({ buttonLink, buttonProps, fromItemGroup, item,
             <LinkedButton
               addClass={`item-button-${orientation} item-link mt-auto`}
               buttonProps={buttonProps}
-              path={fromItemGroup ? { pathname: `${buttonLink}`, query: { id: `${id}` } } : buttonLink}
+              path={{
+                pathname: buttonLink,
+                query: fromItemGroup ? { id } : { id, name }
```

Comment:
needed to pass a query to the browse page items so we can show the name correctly on the new request page and pass the id to the initialize and/or create functions.